src/monodomain_equations_routines.f90

```diff
@@ -1278,15 +1278,15 @@ SUBROUTINE MONODOMAIN_PRE_SOLVE(CONTROL_LOOP,SOLVER,ERR,ERROR,*)
     IF(ASSOCIATED(CONTROL_LOOP%PROBLEM)) THEN
```

Comment:
monodomain_equations_routines.f90. This was originally put in by Sander Land and isn't in the correct form. The mono and bi domain equations are now in biodomain_equations_routines.f90. Add new code to this file as this one will likely be removed one rainy day.